diagnostics/conf/routes

```diff
@@ -14,3 +14,4 @@ GET        /counts.gif                      controllers.DiagnosticsController.an
 GET        /robots.txt                      controllers.Assets.at(path="/public", file="robots.txt")
 
 POST       /css                             controllers.DiagnosticsController.css
+OPTIONS    /css                             controllers.DiagnosticsController.cssOptions
```

Comment:
Could this just be `/*` and cacth any `OPTIONS` request? (sorry if this just shows I have no idea how OPTIONS works)...